lib/aur-build

```diff
@@ -222,11 +225,24 @@ if (( chroot )); then
     printf 'Running aur-chroot %s\n' >&2 "${args[*]}"
     aur chroot "${args[@]}"
 else
+    # makepkg -s does not support setting pacman.conf (#)
```

Comment:
Did you want to link to a github issue here?

Reply:
I thought of filing an issue first, but I went straight to the PR. Removed the #